From ef2f9b921c059fcc768599a5e7d02f04abbec0e7 Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Thu, 20 Dec 2018 16:51:05 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808 move adc key label to specific
 board dts

different board have different key definition, so move adc key label
to specific board dts.

Change-Id: I71da316c639ca0fd063c1ec5c2e303f1a824ed71
Signed-off-by: Lin Huang <hl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts | 14 ++++++++++++++
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi    | 14 +-------------
 2 files changed, 15 insertions(+), 13 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
index 572221f11c3c..33d3885b5474 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
@@ -24,6 +24,20 @@
 	};
 };
 
+&adc_key {
+	vol-down-key {
+		linux,code = <KEY_VOLUMEDOWN>;
+		label = "volume down";
+		press-threshold-microvolt = <300000>;
+	};
+
+	vol-up-key {
+		linux,code = <KEY_VOLUMEUP>;
+		label = "volume up";
+		press-threshold-microvolt = <18000>;
+	};
+};
+
 &display_subsystem {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index 122801937ba7..910a185d6b57 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -12,25 +12,13 @@
 	model = "Rockchip RK1808 EVB";
 	compatible = "rockchip,rk1808-evb", "rockchip,rk1808";
 
-	adc-keys {
+	adc_key: adc-keys {
 		compatible = "adc-keys";
 		autorepeat;
 		io-channels = <&saradc 2>;
 		io-channel-names = "buttons";
 		keyup-threshold-microvolt = <1800000>;
 		poll-interval = <100>;
-
-		vol-down-key {
-			linux,code = <KEY_VOLUMEDOWN>;
-			label = "volume down";
-			press-threshold-microvolt = <300000>;
-		};
-
-		vol-up-key {
-			linux,code = <KEY_VOLUMEUP>;
-			label = "volume up";
-			press-threshold-microvolt = <18000>;
-		};
 	};
 
 	backlight: backlight {
-- 
2.35.3

